.circleci/config.yml

```diff
@@ -31,20 +43,29 @@ jobs:
     steps:
     - checkout
 
-    - run: |
-        wget -q $(curl -sS https://api.github.com/repos/giantswarm/architect/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
-        chmod +x ./architect
-        ./architect version
-
-    - run: ./architect build
+    - run:
+        name: Install architect (except when in cronjob execution)
+        command: |
+            if [ "${CIRCLE_USERNAME}" != "" ]; then
+                wget -q $(curl -sS -H "Authorization: token $RELEASE_TOKEN" https://api.github.com/repos/giantswarm/architect/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
+                chmod +x ./architect
+                ./architect version
+            fi
+
+    - run:
+        name: architect build (except when in cronjob execution)
+        command: test -x ./architect && ./architect build
```

Comment:
Why do we need the `test` here?

Reply:
In the step before, we only download architect if in the push/PR context (`CIRCLE_USERNAME != ""`), not in the cronjob context.

I figured the easiest way to pull this through to subsequent steps would be to use the existence of the architect binary.